A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IdentPage.js

```diff
@@ -23,19 +23,19 @@ class CDUIdentPage {
         mcdu.page.Current = mcdu.page.IdentPage;
         mcdu.activeSystem = 'FMGC';
         mcdu.setTemplate([
-            ["A320-200"],
+            ["A320-251N"],
             ["\xa0ENG"],
-            ["LEAP 1A-26[color]green"],
+            ["LEAP-1A26[color]green"],
             ["\xa0ACTIVE NAV DATA BASE"],
-            ["\xa0" + (date.length === 13 ? date[3] + date[4] + date[0] + date[1] + date[2] + "-" + date[8] + date[9] + date[5] + date[6] + date[7] : date) + "[color]cyan", "AIRAC[color]green"],
+            ["\xa0" + (date.length === 13 ? date[3] + date[4] + date[0] + date[1] + date[2] + "-" + date[8] + date[9] + date[5] + date[6] + date[7] : date) + "[color]cyan", "FBW2014001[color]green"],
             ["\xa0SECOND NAV DATA BASE"],
-            ["{small}{04MAY-04JUL{end}[color]cyan"],
-            ["", "STORED"],
-            ["", "{green}10{end}{small}RTES\xa0{end}{green}10{end}{small}RWYS{end}"],
-            ["CHG CODE", "[b-text]{big}{green}20{end}{end}WPTS\xa0{big}{green}20{end}{end}NAVS"],
-            ["{small}[  ]{end}[color]cyan", "DELETE ALL}[color]cyan"],
-            ["IDLE/PERF", "SOFTWARE"],
-            ["+0.0/+0.0[color]green", "STATUS/XLOAD>"]
+            ["{small}{3DEC-30DEC{end}[color]inop"],
```

Comment:
should be 2101

Reply:
The current nav cycle in MSFS is 2014 not 2101.

Reply:
actually, just copy the code from the ACTIVE NAV DATA BASE, it'll show the correct date

Reply:
people will get confused if I do that